sdk/tests/resources/tex-input-validation.js

```diff
@@ -394,4 +429,98 @@ for (var ii = 0; ii < testCases.length; ++ii) {
   testCopyFromInternalFBO(testCases[ii]);
 }
 
+if (contextVersion > 1) {
+// Create new texture for testing api of WebGL 2.0.
+shouldBeNonNull("tex = gl.createTexture()");
+gl.bindTexture(gl.TEXTURE_3D, tex);
+wtu.glErrorShouldBe(gl, gl.NO_ERROR);
+
+debug("");
+debug("Checking TexImage3D: a set of inputs that are valid in GL but invalid in WebGL");
```

Comment:
Yes, this should aim to find out valid parameters in GL but invalid in GLES. In my opinion, the rules are the same for tex[Sub]Image3D both in GLES and WebGL. So those invalid in GLES are also invalid in WebGL.